client/state/selectors/get-user-setting.ts

```diff
@@ -11,7 +11,7 @@ export default function getUserSetting(
 	state: AppState,
 	settingName: string
 ): UserSettingValue | null {
-	const { settings, unsavedSettings } = state.userSettings;
+	const { settings, unsavedSettings } = state.userSettings ?? {};
```

Comment:
A bunch of unit tests broke when userSettings is `null`, this is fixes those broken tests.